debian7/Dockerfile

```diff
@@ -2,10 +2,12 @@ FROM debian/eol:wheezy
 ARG DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update \
-  && apt-get install -y build-essential curl bzip2 libz-dev
+  && apt-get install -y build-essential bzip2 curl libz-dev texinfo
```

Comment:
Could use `wget` here if it's easier.

Reply:
`wget` is definitely easier, I think I will change everything over to that.